### PR TITLE
Internal Federation Documentation for new API

### PR DESCRIPTION
**Pull request by zhvolosh**

### CODE REVIEW

**Comment by Lauragra** [comment]
@zhvolosh , please reach out to me internally regarding this PR. I can point you to the steps to get set up to be a contributor and information about our content workflow. Thanks.

File: api-reference/beta/resources/federatedIdpMfaBehavior.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+
+---
+title: " federatedIdpMfaBehavior resource type"
+description: ""
```
Comment (Justinha) [inline api-reference/beta/resources/federatedIdpMfaBehavior.md:4]: Explains new Authentication, single sign-on, and MFA

**Comment by Lauragra** [comment]
@zhvolosh , please follow the Get started steps to set yourself up to be a contributor to the documentation. Is your PR ready for a content review? If so, please set the **ready for content review** label (you need to be a member of the contributors team first). The schema associated with this PR is already live so we need to get the docs reviewed and published. Thanks.

**Comment by FaithOmbongi** [comment]
Hi @zhvolosh - I believe this PR is now redundant because the APIs are live. Closing this PR.

**Pull request closed without merging**